github-templates/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+(brief description / PR title)
+
+Status: **Ready for Review** or **Open for Visibility**
```

Comment:
+1 to continuity. There is some disparity between PR templates on CST and Product, and I'm hoping we can have a generic template here that can cover the majority of use cases for all of Mobify engineering. Of course any team can veto this template and use their own (or parts of it) if it makes more sense for them.

I'll try and integrate for a second pass at this.